### PR TITLE
Feature(DTSPB-999): Upgrade crossbrowser dependencies

### PR DESCRIPTION
**Pull request by Rob-Doherty**
### JIRA link (if applicable) ###

https://tools.hmcts.net/jira/browse/DTSPB-1000

### Change description ###

1. Upgrade crossbrowser & e2e dependencies to recent stable versions:
- CodeceptJS v2.6.11
- Puppeteer v5.4.1
- Webdriver v6.10.5
- mocha-multi v1.1.3
- mocha-jenkins-reporter v0.4.5

2. Refactor saucelabs.conf.js to W3C standard for webdriver capabilities

3. Update supportedBrowsers.js list to latest versions

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

File: test/end-to-end/helpers/JSWait.js [hunk 1]
```diff
@@ -64,26 +64,26 @@ class JSWait extends codecept_helper {
...
         } else {
-            const browserName = this.helpers.WebDriverIO.config.browser;
+            const browserName = this.helpers.WebDriver.config.browser;
 
             // browserName !== 'internet explorer' &&  removed
             if (browserName !== 'MicrosoftEdge') {
-                await helper.browser.waitForVisible('#addressLine1', 5000, true); // true - means wait for element to be Invisible!
-                await helper.browser.click('.govuk-details__summary-text');
-                await helper.browser.waitForVisible('#addressLine1', 5000, false);
+                // await helper.waitForVisible('#addressLine1', 5000);
```
Comment (Rob-Doherty) [inline test/end-to-end/helpers/JSWait.js:71]: to be removed if not needed.

Comment (Rob-Doherty) [reply test/end-to-end/helpers/JSWait.js:71]: uncommented

**Review by vinceholt — APPROVED** [review]

File: Jenkinsfile_nightly [hunk 2]
```diff
@@ -50,8 +50,7 @@ static LinkedHashMap<String, Object> secret(String secretName, String envVar) {
 
 withNightlyPipeline("nodejs", product, component) {
     env.TEST_URL = params.URL_TO_TEST
-    env.TEST_E2E_URL = params.URL_TO_TEST
-    env.PATH_TO_TEST_FILES =  params.test_path
+    env.FE_E2E_TEST_PATH_TO_RUN =  params.test_path
```
Comment (Rob-Doherty) [inline Jenkinsfile_nightly:53]: doublecheck with Phani that this isn't going to overlap his changes

Comment (Rob-Doherty) [reply Jenkinsfile_nightly:53]: checked and it's not a problem as Phani is focussed on a different repo (probate-caveat-fronend)

**Review by arun-hmcts — APPROVED** [review]

**Review by mounikahmcts — APPROVED** [review]

**Review by Ferdi-Cocelli — APPROVED** [review]

**Pull request merged**